src/constants/constants.ts

```diff
@@ -4,3 +4,10 @@ export const GITHUB_LINK = 'https://github.com/msdio/stackticon';
 
 export const DEPENDENCY = 'dependencies';
 export const DEV_DEPENDENCY = 'devDependencies';
+
+export type CopyOptionType = 'copy for readme' | 'copy link only' | 'copy img tag';
```

Comment:
How about
```suggestion
type MakeObjectValueToUnion<T> = T[keyof T];

export const COPY_OPTIONS = {
  COPY_README: 'copy for readme',
  COPY_LINK: 'copy link only',
  COPY_TAG: 'copy img tag',
} as const;

export type CopyOptionType = MakeObjectValueToUnion<typeof COPY_OPTIONS>;

```
this?
## Why?
I think, To add a new element to CopyOptionType you need to add it to both CopyOptionType and COPY_OPTIONS.

Therefore, I suggest to use a generic type called MakeObjectValueToUnion, 
When new content is added to COPY_OPTIONS. CopyOptionType is also updated.

## IF..
Please let me know if you separated the type for some other reason.

Reply:
Good point! I will make `type MakeObjectValueToUnion<T> = T[keyof T];` type in types > common, since it could be used  many times in future.

Reply:
> Good point! I will make `type MakeObjectValueToUnion<T> = T[keyof T];` type in types > common, since it could be used many times in future.

nice 😉👍